node/core/dispute-coordinator/src/participation/queues/mod.rs

```diff
@@ -67,7 +67,7 @@ pub struct ParticipationRequest {
 }
 
 /// Whether a `ParticipationRequest` should be put on best-effort or the priority queue.
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
```

Comment:
Just curious - why we need `PartialEq` implemented for this enum?

Reply:
Good catch! I was using that to condense some logic about whether to request participation, but that logic was removed earlier in the process anyways.